src/main/java/org/generationcp/middleware/dao/GermplasmListDataDAO.java

```diff
@@ -103,6 +108,42 @@ public long countByListId(final Integer id) {
 		return ((BigInteger) query.uniqueResult()).longValue();
 	}
 
+	public Map<Integer, List<GermplasmListData>> getGermplasmDataListMapByListIds(final List<Integer> listIds) {
+		try {
+
+			final Criteria criteria = this.getSession().createCriteria(GermplasmListData.class);
+			criteria.createAlias(
```

Comment:
Using class constants within the Hibernate query makes it hard to understand. Please just use the text as is (as applicable)

Reply:
and yes it will likely cause duplicated text warnings but query readability should take priority